library_generation/generate_pr_description.py

```diff
@@ -17,54 +17,51 @@
 import shutil
 from typing import Dict
 from git import Commit, Repo
-from library_generation.model.generation_config import GenerationConfig
+
+from library_generation.model.config_change import ConfigChange
 from library_generation.utils.proto_path_utils import find_versioned_proto_path
 from library_generation.utils.commit_message_formatter import (
     format_commit_message,
-    commit_link,
+    format_repo_level_change,
 )
 from library_generation.utils.commit_message_formatter import wrap_override_commit
 
 EMPTY_MESSAGE = ""
 
 
 def generate_pr_descriptions(
-    config: GenerationConfig,
-    baseline_commit: str,
+    config_change: ConfigChange,
     description_path: str,
     repo_url: str = "https://github.com/googleapis/googleapis.git",
 ) -> None:
     """
-    Generate pull request description from baseline_commit (exclusive) to the
-    googleapis commit (inclusive) in the given generation config.
+    Generate pull request description from configuration comparison result.
```

Comment:
This comment change nicely describes how the pr_description generation will be changed after merging this PR. Could you please add a similar note in the PR description? I could not immediately understand what this PR is about without reading this comment and the implementation changes in `commit_mesage_formatter.py`.

Reply:
Done.